Backend/test_project/order/urls.py

```diff
@@ -9,6 +9,13 @@
 
 router = routers.SimpleRouter()
 
+router.register('product', OrderProductAPIViewSet)
 router.register('', OrderAPIViewSet)
 
+
+urlpatterns = [
+    path("", include(router.urls)),
+]
+
 urlpatterns += router.urls
```

Comment:
You can remove this:
urlpatterns = []
urlpatterns += router.urls
Or this:
urlpatterns = [
    path("", include(router.urls)),
]